packages/core/src/fonts.css

```diff
@@ -106,10 +93,8 @@
 	font-style: normal;
 	font-weight: 700;
 	src: local('Open Sans Bold'), local('OpenSans-Bold'),
-		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.bold.latin-ext.woff2)
-			format('woff2');
-	unicode-range: U+0100-024f, U+0259, U+1-1eff, U+2020, U+20a0-20ab, U+20ad-20cf,
-		U+2113, U+2c60-2c7f, U+A720-A7FF;
+		url(//d5wfroyti11sa.cloudfront.net/prod/fonts/opensans/201902081238.bold.latin-ext.woff2) format('woff2');
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Quote this URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aquoted-url)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this protocol-relative URL. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aprotocol-relative-url)